pkg/codegen/go/gen_program.go

```diff
@@ -582,6 +582,10 @@ func GenerateProjectFiles(project workspace.Project, program *pcl.Program,
 			vPath = fmt.Sprintf("/v%d", p.Version.Major)
 		}
 		packageName := fmt.Sprintf("github.com/pulumi/pulumi-%s/sdk%s/go/%s", p.Name, vPath, p.Name)
+		if p.SupportPack {
+			// then the default is that we no longer nest source files under the go subdirectory
+			packageName = fmt.Sprintf("github.com/pulumi/pulumi-%s/sdk/go%s", p.Name, vPath)
```

Comment:
Doesn't this still need p.Name at the end (e.g. go/v2/simple)

Reply:
This is just the package name used in `go.mod` not the base import path